src/async.c

```diff
@@ -276,39 +289,25 @@ mpd_async_io(struct mpd_async *async, enum mpd_async_event events)
 	return true;
 }
 
-bool
-mpd_async_send_command_v(struct mpd_async *async, const char *command,
-			 va_list args)
+static bool
+quote_vargs(struct mpd_buffer *buffer, char *start, char **end_pos,
```

Comment:
What does this function do?
Documentation missing!

Reply:
fixed in https://github.com/MusicPlayerDaemon/libmpdclient/pull/53/commits/14a9563740b652aaeae007ccc93cadda44211e9c